### DISCUSSION TITLE
get all components and tags from entity?

### DISCUSSION THREAD

**Discussion by ludek17052** [Quillraven/Fleks#132] [General]
Can I get all components and tags from entity?

Now I can use: playerEntity.get(PlayerComponent)
to get one component.

I would like to ask if I can get list of all components or tags?
If it is not possible can I ask you to add this function?
Thank you.

**Comment by Quillraven** [comment]
Check Out the wiki of this repository. There is a functionality called snapshot to achieve what you are trying to do.

You can get the snapshot of an entire world or a snapshot of a specific entity. It will contain it's components and tags.

world.snapshotOf(entity)

**Comment by ludek17052** [comment]
OK. Thank you. I will try it.

**Reply by Quillraven** [reply]
No problem! Please mark the correct answer when you verified the solution. Gives a better overview in the discussion area 🙂

**Reply by ludek17052** [reply]
OK. I marked the answer.
